common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -451,6 +454,48 @@ class @Problem
       element.CodeMirror.save() if element.CodeMirror.save
     @answers = @inputs.serialize()
 
+  checkAnswersAndCheckButton: (bind=false) =>
+    answered = true
+
+    @el.find("input:text").each (i, v) =>
```

Comment:
Could you use longer/more descriptive variable names here? It makes the code easier to read, and it would mean you wouldn't have to resort to using `vv` to distinguish them.